config/settings.py

```diff
@@ -69,8 +71,43 @@ def __init__(self, path: str):
             self.path = path
 
     class Server:
-        def __init__(self, user_ip: str, private_key: str, endpoint_ip: str, endpoint_port: str):
+        def __init__(self, user_ip: str, private_key: str, public_key:str, endpoint_ip: str, endpoint_port: str):
             self.user_ip = user_ip
             self.private_key = private_key
+            self.public_key = public_key
             self.endpoint_ip = endpoint_ip
             self.endpoint_port = endpoint_port
+        
+        def create_wg_server_base(self):
+            return f"""[Interface]
+Address = {self.user_ip}.1/24
+ListenPort = {self.endpoint_port}
+PrivateKey = {self.private_key}
+"""
+
+        def setup_server_keys(self):
+            server_private_key = private_key()
+            server_public_key = public_key(server_private_key)
+            self.privatekey = server_private_key
+            self.public_key = server_public_key
+
+        def create_wg_server_config(path, base):
+            try:
+                wg_file = open(path, "w")
+                wg_file.write(base)
+                wg_file.close()
+                return 0
```

Comment:
```suggestion
                return True
```
returning `0 (== False)` may be unclear. consider returning True if operation was successful